Automattic/sensei
# Include Select2 library as NPM dependency

Fixes #2180 & #2181.

## Testing
- Run `gulp` and ensure the following files are copied to the _assets/vendor/select2_ folder:
  - select2.full.js
  - select2.full.min.js
  - select2.min.css
- Check the different Sensei pages (courses, lessons etc.) to ensure the drop-downs are styled and behave as before.
- Add the following line to your _wpconfig.php_ file - `define( 'SCRIPT_DEBUG', true );`
- Check that the _select2.full.js_ file is loaded instead of the minified version.
- Check the different Sensei pages (courses, lessons etc.) to ensure the drop-downs are styled and behave as before.
